File: 389-find-the-difference/find-the-difference.cpp
class Solution {
public:
    char findTheDifference(string s, string t) {
       int sum=0;
       for(auto i:t){
        sum+=(i-'a');
       }
       for(auto i:s){
        sum-=(i-'a');
       }
       
       return (sum+'a');
    }
};

File: 389-find-the-difference/find-the-difference.js
/**
 * @param {string} s
 * @param {string} t
 * @return {character}
 */
var findTheDifference = function(s, t) {
    var n=0;
    var m=0;
    for(const i of s){
        n+=i.charCodeAt(0);
    }
    for(const i of t){
        m+=i.charCodeAt(0);
    }
    return String.fromCharCode(m - n);;

};